pkg/openSearch/openSearchClient/client.go

```diff
@@ -63,8 +63,11 @@ func (c *Client) Search(indexName string, requestBody []byte) (responseBody []by
 		return nil, err
 	}
 
-	log.Trace().Str("src", "opensearch").Msgf("search response - statusCode:'%d' json:'%s'",
-		searchResponse.Inspect().Response.StatusCode, string(result))
+	log.Trace().
+		Msgf(
```

Comment:
status code and and body could go to separate fields instead of remaining in `Msgf`

Reply:
Msgf can remain unchanged, see [comment](https://github.com/greenbone/opensight-golang-libraries/pull/95#discussion_r1696699393).

It probably still makes sense to add the status code as separate field of the structured log.